native/core/src/parquet/parquet_exec.rs

```diff
@@ -61,7 +61,12 @@ pub(crate) fn init_datasource_exec(
     data_filters: Option<Vec<Arc<dyn PhysicalExpr>>>,
     session_timezone: &str,
 ) -> Result<Arc<DataSourceExec>, ExecutionError> {
-    let (table_parquet_options, spark_parquet_options) = get_options(session_timezone);
+    let has_data_filters = match &data_filters {
+        Some(filters) => !filters.is_empty(),
+        None => false,
+    };
```

Comment:
This may be redundant .. if there are no filters then we probably don't need to change the DataFusion pushdown options

Reply:
I removed this code